ibis/expr/types.py

```diff
@@ -63,6 +64,13 @@ def _repr(self, memo: 'Optional[FormatMemo]' = None):
 
         return ExprFormatter(self, memo=memo).get_result()
 
+    @staticmethod
+    def _normalize(arg):
```

Comment:
What do you think about a dispatcher for literal values that gets called in the `ops.Literal` constructor?

Something like

```python

_normalize = Dispatcher("_normalize")

@_normalize.register(dt.Float64, (int, float))
def _normalize(type, value) -> float:
    return float(value)
```

Then this would be called wherever the `value` is passed into `ops.Literal`.

Reply:
oooh, yeah, I like that a lot